### PR TITLE
For #93: Bump qulice and remove suppression of IndentationCheck

### PR DESCRIPTION
**Pull request by andreoss**
This is for #93

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#157](https://github.com/llorllale/cactoos-matchers/issues/157) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/6e63cd8f-93b9-48f7-a722-8280c4d75acb); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/llorllale/cactoos/matchers/MatcherEnvelopeTest.java [hunk 1]
```diff
@@ -104,8 +104,9 @@ public void decoratesDescribeMismatchSafely() {
                     "must delegate describeMismatchSafely to encapsulated matcher",
                     MatcherEnvelopeTest.TEST_INTEGER,
                     new MatcherEnvelopeChild<>(
-                        (item) -> false,
-                        (description) -> { },
+                        item -> false,
+                        description -> {
+                        },
```
Comment (victornoel) [inline src/test/java/org/llorllale/cactoos/matchers/MatcherEnvelopeTest.java:109]: @andreoss can't this stay on one line? maybe by removing the space between `{` and `}`?

**Comment by andreoss** [comment]
> @andreoss very nice, just one small comment

@victornoel Fixed

**Comment by victornoel** [comment]
@andreoss thx :)

@paulodamaso lgtm

**Comment by victornoel** [comment]
@paulodamaso looks good to me

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (13 days), architects (@paulodamaso, @victornoel[/z](https://www.0crat.com/u/victornoel)) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Pull request merged**